test/mocha.opts

```diff
@@ -1,2 +1,2 @@
---timeout 20000
+--timeout 25000
```

Comment:
When running `multibuild.spec.ts` against a NUC and an RPi3 (to test build secrets and multiple architectures), the RPi3 needed an extra 5 seconds...